astronomer/airflow/version_check/templates/update-available.html

```diff
@@ -27,6 +27,12 @@
     </div>
   {% endif %}
 
+  {% if cea_yanked_warning %}
+    <div class="alert alert-warning ac-update-notice">
+      <p><strong>{{ cea_yanked_warning | safe }}</strong></p>
```

Comment:
```suggestion
      <p><strong>{{ cea_yanked_warning}}</strong></p>
```

With the link gone, we can rip this out.

Reply:
Changed it